packages/styles/forms.css

```diff
@@ -20,6 +20,11 @@
   --field-icon-inactive-color: var(--gray-90);
   --field-icon-active-color: rgba(60, 122, 174, 0.25);
   --field-icon-error-active-color: rgba(217, 50, 81, 0.25);
+  --field-icon-indeterminate-color: var(--accent-primary);
+  --field-icon-indeterminate-disabled-color: var(
+    --accent-primary-dsiabled,
+    #78a6d8
+  );
```

Comment:
I don't think new colors are necessary. This should be using the existing `--field-icon-xxx` colors for the appropriate states.

Reply:
Thanks, removed